stock_picking_mgmt_weight/models/stock_move.py

```diff
@@ -0,0 +1,62 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api, _, tools
+from datetime import date, datetime
```

Comment:
Please confirm if these imports are unnecesary and can be removed:
```suggestion
from odoo import models, fields, api, _
```